F1WM/Repositories/DriversRepository.cs

```diff
@@ -0,0 +1,44 @@
+using AutoMapper;
+using F1WM.ApiModel;
+using F1WM.DatabaseModel;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace F1WM.Repositories
+{
+	public class DriversRepository : RepositoryBase, IDriversRepository
+	{
+		private readonly IMapper mapper;
+
+		public async Task<Drivers> GetDrivers(string letter)
+		{
+			await SetDbEncoding();
+
+			Drivers result = new Drivers();
+
+			result.DriversList = await mapper.ProjectTo<DriverSummary>(
```

Comment:
The formatting of this expression confuses me a bit. Please add indentation to mark what's being done on what collection.